src/net/sourceforge/kolmafia/request/WitchessRequest.java

```diff
@@ -15,7 +15,8 @@ public WitchessRequest() {
 
   @Override
   public void run() {
-    if (!KoLCharacter.inLegacyOfLoathing()
+    if (!(KoLCharacter.inLegacyOfLoathing()
+            && Preferences.getBoolean("replicaWitchessSetAvailable"))
         && !StandardRequest.isAllowed(RestrictedItemType.ITEMS, "Witchess Set")) {
       KoLmafia.updateDisplay("Witchess is too old to use in your current path.");
```

Comment:
I guess. I'm not going to insist, but there could be different messages to distinguish "not in standard" from "in LoL but have not bought a replica Witchess set yest".